PL2/PL2-gui/src/main/java/nl/tudelft/pl2016gr2/gui/view/graph/DrawComparedGraphs.java

```diff
@@ -154,37 +170,37 @@ private static void drawGraph(Pane pane, ArrayList<NodePosition> graph, int star
       }
     }
     drawNode(pane, circleMap, levelNodes, curLevel, startLevel);
-    repositionOverlappingEdges(graph, startIndex, endIndex, circleMap);
-    drawEdges(pane, graph, startIndex, endIndex, circleMap);
+    repositionOverlappingEdges(graphOrder, startIndex, endIndex, circleMap);
+    drawEdges(pane, graphOrder, graph, startIndex, endIndex, circleMap);
   }
 
   /**
    * Calculates the starting index (where to start in the graph with drawing). Lowers the start
    * index by 20 to keep some margin at the left of the screen (so the edges are drawn correctly)
    *
-   * @param graph      the graph.
+   * @param graphOrder the graph node order.
    * @param startLevel the start level.
    * @return the start index.
    */
-  private static int calculateStartIndex(ArrayList<NodePosition> graph, int startLevel) {
+  private static int calculateStartIndex(ArrayList<NodePosition> graphOrder, int startLevel) {
     int actualStartLevel = startLevel - OFFSCREEN_DRAWN_LEVELS;
```

Comment:
The comment says `20`, but the value of `OFFSCREEN_DRAWN_LEVELS` is `10`. Let the comment link to the constant with `{@link #OFFSCREEN_DRAWN_LEVELS}`